quota-notification/src/main/java/functions/ConfigureAppAlertHelper.java

```diff
@@ -276,7 +288,8 @@ private static String createOrUpdateNotificationChannel(AppAlert appAlert){
         notificationChannel = updateNotificationChannel(client, appAlert);
       }
     } catch (IOException e) {
-      logger.severe("Can't create Notification channel "+e.toString());
+      MDC.put("severity", "CRITICAL");
```

Comment:
MDC is to add the severity field back in cloud logging after switching to using log back https://medium.com/google-cloud/java-logging-on-cloud-run-with-stackdriver-9786d6fdbe17